todos-lambda/scripts/build-and-package.sh

```diff
@@ -13,20 +13,7 @@
 ##
 ##===----------------------------------------------------------------------===##
 
-set -eu
+here=$(dirname "$0")
 
-base=$(pwd)
```

Comment:
One thing we add to our build scripts is a check that the user is on ARM64

Reply:
What's your actual check?

Reply:
In our case we enforce AMD64, due to us having had some issues specific to Lambda runtime on ARM64:

```sh
if [ "$(uname -m)" = "arm64" ]; then
  echo "Error: Unsupported architecture. Please build from an AMD64 CPU or use the GitHub CI"
  exit 1
fi
```

Reply:
Added architecture check